geolocator_platform_interface/lib/src/errors/approximate_location_not_supported_exception.dart

```
@@ -0,0 +1,21 @@
+/// An exception thrown when users using iOS 13 or below try to execute the
+/// 'requestTemporaryFullAccuracy' method.
+///
+/// Since Approximate location only supports iOS 14 or above, the exception is
+/// only thrown when using iOS 13 or below.
```

The documentation only mentions the use of iOS but what happens when you use another platform (Android or Web)?

I think in those cases we should throw the same exception (which is the responsibility of the `geolocator` package and should not be implemented here) but we should update the documentation here to mention other platforms (or make it clear that this exception is thrown on anything other than iOS 14).